daemon/logger/fluentd/fluentd.go

```diff
@@ -111,9 +111,9 @@ func New(info logger.Info) (logger.Logger, error) {
 		maxRetries = int(mr64)
 	}
 
-	asyncConnect := false
+	async := false
 	if info.Config[asyncConnectKey] != "" {
-		if asyncConnect, err = strconv.ParseBool(info.Config[asyncConnectKey]); err != nil {
+		if async, err = strconv.ParseBool(info.Config[asyncConnectKey]); err != nil {
```

Comment:
IMO it's better to do:
* add `asyncKey` (fluentd-async) to enable `async` option
* warn that `asyncConnectKey` is now not recommended and will be removed later, and also enable `async` if `asyncConnectKey` is specified (and unless `asyncKey` is disabled explicitly)

Reply:
Ah, right; my main goal with this PR was to keep the dependency up-to-date with upstream fixes, without changing the UX / options.

I'd be ok with adding a `fluentd-async` and marking `fluentd-async-connect` as deprecated (and perhaps adding an error or warning if both are specified?)

Probably better for a follow-up 🤗

Reply:
Let me rename the variable back in this PR though